source/hic_hal/nxp/lpc11u35/gpio.c

```diff
@@ -82,6 +82,7 @@ void gpio_init(void)
     // enable clock for GPIO port 0
     LPC_SYSCON->SYSAHBCLKCTRL |= (1UL << 6);
     // configure GPIO-LED as output
+#if !defined(BLUENINJA_SB)
```

Comment:
This is common code so it shouldn't contain board specific logic.  There will need to be feature switch